### PR TITLE
Add the standard Symfony console to the project

### LINKED ISSUE #979: bin/opencfp cache:clear throws exception
# Steps required to reproduce the problem

1. Set up project, install dependencies, etc.
2. Run `bin/opencfp cache:clear`

# Expected Result

* Expect all caches to be cleared.

# Actual Result

```bash
mdwheele@28d9116def76:/usr/src/app$ bin/opencfp cache:clear

OpenCFP
=======

Clearing caches
---------------

  * /usr/src/app/cache/development (214 files and directories)

                                                                                                                        
 [OK] Cleared caches.                                                                                                   
                                                                                                                        


Warning: Uncaught Symfony\Component\Debug\Exception\ContextErrorException: Warning: require(/usr/src/app/cache/development/ContainerCxf10iq/getSwiftmailer_EmailSender_ListenerService.php): failed to open stream: No such file or directory in /usr/src/app/cache/development/ContainerCxf10iq/OpenCFPDevelopmentDebugProjectContainer.php:543
Stack trace:
#0 /usr/src/app/cache/development/ContainerCxf10iq/OpenCFPDevelopmentDebugProjectContainer.php(543): require()
#1 /usr/src/app/cache/development/ContainerCxf10iq/OpenCFPDevelopmentDebugProjectContainer.php(948): ContainerCxf10iq\OpenCFPDevelopmentDebugProjectContainer->load('/usr/src/app/ca...')
#2 /usr/src/app/vendor/symfony/event-dispatcher/EventDispatcher.php(229): ContainerCxf10iq\OpenCFPDevelopmentDebugProjectContainer->ContainerCxf10iq\{closure}()
#3 /usr/src/app/vendor/symfony/event-dispatcher/EventDispatcher.php(61): Symfony\Component\EventDispatcher\EventDispatcher->sortListeners('console.termina...')
#4 /usr/src/app/vendor/symfony/event-dispatcher/ContainerAwareEve in /usr/src/app/cache/development/ContainerCxf10iq/OpenCFPDevelopmentDebugProjectContainer.php on line 543

Fatal error: ContainerCxf10iq\OpenCFPDevelopmentDebugProjectContainer::load(): Failed opening required '/usr/src/app/cache/development/ContainerCxf10iq/getSwiftmailer_EmailSender_ListenerService.php' (include_path='.:/usr/local/lib/php') in /usr/src/app/cache/development/ContainerCxf10iq/OpenCFPDevelopmentDebugProjectContainer.php on line 543

In OpenCFPDevelopmentDebugProjectContainer.php line 543:
                                                                                                                                                                                                           
  [Symfony\Component\Debug\Exception\FatalErrorException]                                                                                                                                                  
  Compile Error: ContainerCxf10iq\OpenCFPDevelopmentDebugProjectContainer::load(): Failed opening required '/usr/src/app/cache/development/ContainerCxf10iq/getSwiftmailer_EmailSender_ListenerService.ph  
  p' (include_path='.:/usr/local/lib/php')                                                                                                                                                                 
                                                                                                                                                                                                           

Exception trace:

cache:clear [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [--env ENV] [--] <command>

```

Not being familiar with Symfony, I assume it's clearing cache and then something else is expecting to require a file that was cleared to do work. I can dig in later. I spent 5 minutes wondering why my settings weren't updating and remembered "Oh yeah... container cache... it's a thing now."

### PR DESCRIPTION
**Pull request by derrabus**
This PR add the standard Symfony console to the project and lets it take over the `cache:clear` command.

Please test if that works out better for you.

The console has some helpful maintenance and debug commands, so it's probably good to have it from a developer perspective. We could merge the console with the `bin/opencfp` console, but I'm not sure if we should flood the end-user with framework-related commands. So it might be good to have to separate console applications.

Fixes #979. (kind of)

### CODE REVIEW

**Comment by localheinz** [comment]
@mdwheele 

Since you were having problems, do you want to take a look as well?

**Comment by mdwheele** [comment]
I do! I can do so a bit later. Can we go ahead and make a call on moving our console commands into this? Personally, I'm :+1:. Not worried about floods of commands as they'll be documented anyway.

**Comment by chartjes** [comment]
Is this PR safe to merge?

File: resources/config/services/console.yml [hunk 1]
```diff
@@ -5,17 +5,14 @@ services:
 
   OpenCFP\Console\Command\:
     resource: '%kernel.project_dir%/classes/Console/Command/*'
-
-  OpenCFP\Console\Command\ClearCacheCommand:
-    arguments:
-      $paths: ['%kernel.cache_dir%']
+    tags:
+      - { name: console.command }
```
Comment (localheinz) [inline resources/config/services/console.yml]: @chartjes @derrabus @mdwheele 

I think with this it should be safe going forward and having the commands automatically registered - at least https://symfony.com/doc/2.6/cookbook/console/commands_as_services.html suggests to do so.

Running

```
$ bin/console
```

yields

```
./bin/console
Symfony 3.4.2 (kernel: OpenCFP, env: development, debug: true)

Usage:
  command [options] [arguments]

Options:
  -h, --help            Display this help message
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi            Force ANSI output
      --no-ansi         Disable ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -e, --env=ENV         The environment name [default: "development"]
      --no-debug        Switches off debug mode
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug

Available commands:
  about                      Displays information about the current project
  help                       Displays help for a command
  list                       Lists commands
 assets
  assets:install             Installs bundles web assets under a public directory
 cache
  cache:clear                Clears the cache
  cache:pool:clear           Clears cache pools
  cache:pool:prune           Prune cache pools
  cache:warmup               Warms up an empty cache
 config
  config:dump-reference      Dumps the default configuration for an extension
 debug
  debug:autowiring           Lists classes/interfaces you can use for autowiring
  debug:config               Dumps the current configuration for an extension
  debug:container            Displays current services for an application
  debug:event-dispatcher     Displays configured listeners for an application
  debug:form                 Displays form type information
  debug:router               Displays current routes for an application
  debug:swiftmailer          Displays current mailers for an application
  debug:translation          Displays translation messages information
  debug:twig                 Shows a list of twig functions, filters, globals and tests
 eloquent
  eloquent:make:seeder       Create a new seeder class
  eloquent:migrate           Executes a migration.
  eloquent:migrate:fresh     Drop all tables and re-run all migrations.
  eloquent:migrate:install   Creates the migration repository.
  eloquent:migrate:make      Creates a new migration file
  eloquent:migrate:refresh   Reset and re-run all migrations
  eloquent:migrate:reset     Rollback all database migrations
  eloquent:migrate:rollback  Rollback the last database migration
  eloquent:migrate:status    Show the status of each migration
  eloquent:seed              Seed the database with records
 lint
  lint:twig                  Lints a template and outputs encountered errors
  lint:xliff                 Lints a XLIFF file and outputs encountered errors
  lint:yaml                  Lints a file and outputs encountered errors
 router
  router:match               Helps debug routes by simulating a path info match
 swiftmailer
  swiftmailer:email:send     Send simple email message
  swiftmailer:spool:send     Sends emails from the spool
 translation
  translation:update         Updates the translation file
 user
  user:create                Creates a new user
  user:demote                Demote an existing user from a role
  user:promote               Promote an existing user to a role
```

💡 Note our commands at the bottom.

**Comment by localheinz** [comment]
Thank you, @derrabus!

**Pull request merged**